typescript/infra/scripts/warp-routes/monitor/monitor-warp-route-balances.ts

```diff
@@ -310,6 +322,18 @@ async function getCoinGeckoApiKey(): Promise<string | undefined> {
   return apiKey;
 }
 
+function getWarpRouteCollateralTokenSymbol(warpCore: WarpCore): string {
+  const collateralToken = warpCore.tokens.find(
+    (token) =>
+      token.isCollateralized() ||
+      token.standard === TokenStandard.EvmHypXERC20Lockbox,
+  );
+  if (!collateralToken) {
```

Comment:
eventually we may have warp routes that are all synthetic, e.g. if someone issues their token natively as a warp route

could we maybe have the logic be "if all token symbols are consistent, use that, otherwise pick the collateral one. If there are multiple collateral symbols and they differ, arbitrarily use the symbol that is first alphabetically"

Reply:
also a comment describing why we need to have special logic would probably be good

Makes me wonder if the registry every gets in weird scenarios here with differring token symbols too